test/web-platform-tests/to-run.yaml

```diff
@@ -258,7 +258,7 @@ Document-characterSet-normalization-2.html: [timeout, Some encodings are not sup
 Document-createEvent-touchevent.window.html: [fail, Pointer events not implemented]
 Document-createEvent.https.html: [fail, We don't support every event interface yet]
 Document-getElementById.html: [fail, We cache IDs in insertion order]
-Element-closest.html: [fail, :has is not supported (by all major browsers as well)]
+Element-closest.html: [fail, test with :invalid pseudo-class is failing]
```

Comment:
1/29 errors in test.
Only failure is that `fieldset.checkValidity()` returns `true` even if the input element within the fieldset is invalid.
Seem to me a bug of jsdom?

Other pseudo-classes e.g. `:not()`, `:has()` are passing as expected.